_blogposts/2024-02-01-release-11-1.mdx

```diff
@@ -0,0 +1,117 @@
+---
+author: rescript-team
+date: "2024-02-29"
+previewImg: /static/blog/compiler_release_11_0.jpg
+title: ReScript 11.1
+badge: release
+description: |
+  Unleashing ReScript from React
+---
+
+A couple of weeks ago, the ReScript team released ReScript 11.0, which laid ground work for a lot of possible improvements to make it easier to interact with the JavaScript ecosystem. It also came with a lot of breaking changes, unfortunately.
```

Comment:
Not sure if we should emphasize those breaking changes that much. :-)

Reply:
Agreed. I don't think we need to say anything about the breaking changes at all, enough that they were mentioned in the v11 post.